main.py

```diff
@@ -337,6 +342,50 @@ def isPaused(self):
 	def isRunning(self):
 		return self._p is not None # FIXME: Check subproc state
 
+class AppVeyor(object):
+	def __init__(self, repo, branch, user):
+		self.gjson = json.loads(requests.get("https://ci.appveyor.com/api/projects/" + user + "/" + repo + "/branch/" + branch).text)
+
+	def GetJobs(self):
+		return self.gjson["build"]["jobs"]
+
+	def GetRelease(self, id):
+		idjson = json.loads(requests.get("https://ci.appveyor.com/api/buildjobs/" + str(id) + "/artifacts").text)
```

Comment:
Avoid call in call.

You should probably move `requests.get("https://ci.appveyor.com/api/buildjobs/" + str(id) + "/artifacts")` into a separate line, which also allows you to check if it was a success or not.